web/packages/teleport/src/Roles/RoleEditor/StandardEditor/AccessRules.tsx

```diff
@@ -112,6 +115,26 @@ const AccessRule = memo(function AccessRule({
         value={verbs}
         onChange={v => setRule({ ...value, verbs: v })}
         rule={precomputed(validation.fields.verbs)}
+      />
+      <FieldInput
+        label="Condition"
```

Comment:
personally i think it's okay to label it this way, but i'd also double check with product team because we try to teach users consistent wording

Reply:
After consulting UX/Product, we decided to rename it to **Filter**.